### ISSUE TITLE
Tabs vs space issue in fixed.py:FixedLabelPartitionClassifier

### ISSUE THREAD

**Issue by simon-m** [scikit-multilearn/scikit-multilearn#67]
Hello,

when importing LabelSpacePartitioningClassifier, I get the following error:
```
[trimmed]
  File "C:\Program Files\WinPython-32bit-3.6.1.0Qt5\python-3.6.1\lib\site-packages\skmultilearn\ensemble\fixed.py", line 29
    self.label_count = y.shape[1]
                                ^
TabError: inconsistent use of tabs and spaces in indentation
```
And it turns out the first and last line of the block below start with tabs instead of spaces:
```
    	"""Assign fixed partition of the label space

    	Mock function, the partition is assigned in the constructor.
```

Best,

Simon

**Comment by ljvmiranda921** [comment]
Hi, from where did you install `scikit-multilearn`? Was it cloned from Github or through `pip install`?

**Comment by simon-m** [comment]
I used pip install.

**Comment by ljvmiranda921** [comment]
Hi @simon-m 

As of now, the bleeding-edge version that can be installed via `git clone` has already solved this problem (reference PR #65 and [commit 6abc196](https://github.com/scikit-multilearn/scikit-multilearn/commit/6abc196f1d66bc5c5c570de0aaa9c0db0be89b48). However, we haven't released this version yet in PyPI that's why you're getting the old error.

I recommend that you install `scikit-multilearn` by cloning this repository, thus:

```bash
$ git clone https://github.com/scikit-multilearn/scikit-multilearn.git
$ cd scikit-multilearn
$ python setup.py
```

Cheers!
